config/v1/types_network.go

```diff
@@ -67,6 +73,36 @@ type ClusterNetworkEntry struct {
 	HostPrefix uint32 `json:"hostPrefix"`
 }
 
+// ExternalIPConfig specifies some IP blocks relevant for the ExternalIP field
+// of a Service resource.
+type ExternalIPConfig struct {
```

Comment:
Need to explain on one of the two fields how they interact. (That is, that automatically-assigned egress IPs are allowed regardless of `policy`.)

Reply:
very good point.